src/test/java/com/ibm/cloud/sdk/core/test/model/DynamicModelSerializationTest.java

```diff
@@ -261,7 +261,7 @@ public void testModelAPString() {
   @Test
   public void testModelAPFooNullTypeToken() {
     ModelAPFooNullTypeToken model = createModelAPFooNullTypeToken();
-    testSerDeser(model, ModelAPFooNullTypeToken.class, true);
+    testSerDeser(model, ModelAPFooNullTypeToken.class, false);
```

Comment:
This change eliminates the need to ever "trim decimals" in `testSerDeser`, so lets just clean that up as part of this PR.  Meaning ... eliminate that parameter and the associated logic from `testSerDeser` and all the call sites.

Reply:
☝️ agreed